idearium-lib/lib/query.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+/**
+ * Assign some defaults to the passed Mongoose query options.
+ * @param {Object} options Mongoose query options.
+ * @return {Object} Returns the options object for Mongoose queries.
+ */
+const getOptions = (options) => {
+
+    const defaults = {
+        filter: {},
+        lean: true,
+        limit: 10,
+        projection: '',
+    };
+
+    Object.assign(defaults, options);
+
+    // We want to return all fields.
```

Comment:
How does this work? You have to pass `false` in `options.projection` to retrieve all fields?